routing/routing.xml

```diff
@@ -549,7 +561,15 @@
 			<select value="-1" t="mtb:scale" v="5"/>
 			<select value="-1" t="mtb:scale" v="6"/>
 
-			<select value="-1" t="vehicle" v="no"/>
+			<select value="1" t="vehicle" v="no"/>
+			<!-- The tag vehicle=no is normally used when mappers
+			want to exclude motor vehicles. I have never seen a case
+			when such a road was actually forbidden for bicycles as
+			well. (Mappers often do not bother to add the bicycle=yes).
+			Example:www.openstreetmap.org/way/159679495  This is
```

Comment:
I was wrong: It is the better idea to let "vehicle=no" as it was. It is only important to check "bicycle=yes" (etc) first, and put "vehicle=no" behind that. I have put it in like this in my updated pull request (that I put online today). 

The compromise (downgrading it later) does not work for the following reason: To make the downgrade work, we have to put it before "surface=asphalt". But in that case, the "bicycle=yes" comes behind the downgrade for "vehicle=no". So the very common case (where we have vehicle=no combined with bicycle=yes) would produce a wrong result: It would get a strong downgrade, even though bicycle=yes is marked. www.openstreetmap.org/way/288180634